lib/std/child_process.zig

```diff
@@ -276,7 +276,7 @@ pub const ChildProcess = struct {
 
         // Windows Async IO requires an initial call to ReadFile before waiting on the handle
         for ([_]u1{ 0, 1 }) |i| {
-            try outs[i].ensureCapacity(bump_amt);
+            try outs[i].ensureTotalCapacity(bump_amt);
```

Comment:
Is this correct? `bump_amt` is used as a delta not a total capacity everywhere else in this file it seems...

Reply:
Yeah, I think you're right. Note that this was incorrect before as well (`ensureCapacity` is an alias for `ensureTotalCapacity`).

Reply:
Actually, since it operates separate ArrayLists on each iteration (`out[0]` and then `out[1]`) it doesn't matter in this case. However, I think it probably should use the same logic as `collectOutputPosix` taking `max_output_bytes` into account:

```zig
const new_capacity = std.math.min(outs[i].items.len + bump_amt, max_output_bytes);
try outs[i].ensureTotalCapacity(new_capacity);
```